### PR TITLE
7 24 News Updates

### PR DESCRIPTION
**Pull request by EKC-OOI**

### CODE REVIEW

File: content/_news - KYW
Comment (seidior) [inline content/_news - KYW:1]: @EKC-OOI Can this file be removed?

File: content/_news/2024-07-26-disability-hub-launch.md
Comment (seidior) [inline content/_news/2024-07-26-disability-hub-launch.md:1]: @EKC-OOI Is this similar or should it replace this? https://github.com/newjersey/innovation.nj.gov/pull/177/files

**Comment by EKC-OOI** [comment]
Sorry, I can't understand what I'm looking at in that format?

**Comment by EKC-OOI** [comment]
@seidior Think I see what the question is...there's a pull request 177 that
has a disability hub item...I would just delete that and go with what I
have to keep it simple.

File: content/_news - KYW
Comment (EKC-OOI) [reply content/_news - KYW:1]: @seidior Yes please. I screwed up the file name and couldn't fix it, so tried to delete it. Not sure if I succeed though.

**Pull request merged**